src/spec/integration/import.spec.coffee

```diff
@@ -1527,3 +1527,62 @@ describe 'Import integration test', ->
 
         done()
       .catch (err) -> done _.prettify(err)
+
+    it 'should update product with multiple update requests', (done) ->
+      client = @importer.client
+      mockProduct = {
+        key: 'mockProduct',
+        name: {
+          en: 'test product'
+        },
+        productType: {
+          typeId: 'product-type',
+          id: @productType.id
+        },
+        slug: {
+          en: 'test-product'
+        },
+        description: {
+          en: 'test description'
+        },
+        masterVariant: {
+          sku: 'var1',
+          key: 'var1'
+        },
+        variants: []
+      }
+      request = {
+        uri: TestHelpers.createService(project_key, 'products').build()
+        method: 'POST'
+        body: mockProduct
+      }
+
+      client.execute(request)
+        .then (res) =>
+          mockUpdateRequests = [{
```

Comment:
I would have gone for `mockUpdateRequests -> updateActions`, but this is fine I guess 🤷‍♂️